### PR TITLE
More stacks fixes

### LINKED ISSUE #15485: Wrenching table parts
(WEB REPORT BY: yackemflam REMOTE: 172.93.109.202:7777)
>Revision (Should be above if you're viewing this from ingame!)

>General description of the issue
Table parts disappear when you wrench them in hand

>What you expected to happen
Table parts to turn into a sheet of their material

>What actually happened
They disappeared into somewhere

>Steps to reproduce if possible
Wrench a table part that is in your hand

### PR DESCRIPTION
**Pull request by DamianX**
Fixes more instances of shit like #15243, specifically for grilles' rods and windows' sheets.
Fixes #15485, turns out spawning things in `src.loc` is a bad idea if `src` can be picked up in hand.
Also adds a `forceMove()` call to `/obj/item/stack/New()` so that stacks are properly merged on creation.

### CODE REVIEW

File: code/game/objects/structures/grille.dm [hunk 1]
```diff
@@ -33,14 +33,14 @@
...
-		getFromPool(/obj/item/stack/rods, get_turf(src)) //One rod set
+		new /obj/item/stack/rods(get_turf(src)) //One rod set
 	else if(health >= (0.25*initial(health)) && broken) //Repair the damage to this bitch
 		broken = 0
 		icon_state = initial(icon_state)
 		density = 1
 	if(health <= 0) //Dead
-		getFromPool(/obj/item/stack/rods, get_turf(src)) //Drop the second set of rods
-		returnToPool(src)
+		new /obj/item/stack/rods(get_turf(src)) //Drop the second set of rods
```
Comment (unid15) [inline code/game/objects/structures/grille.dm:42]: Why remove getFromPool()?

Comment (DamianX) [reply code/game/objects/structures/grille.dm:42]: it just works

Comment (unid15) [reply code/game/objects/structures/grille.dm:42]: Does this fix any issues? If not, then this does nothing except very slightly worsen performance

Comment (DamianX) [reply code/game/objects/structures/grille.dm:42]: When I first started looking into this merely changing from getFromPool to new solved the issue, I don't know if it's still relevant after the other fixes, and I was under the impression that pooling didn't increase the performance anyway.

Comment (Exxion) [reply code/game/objects/structures/grille.dm:42]: By my understanding, the current belief is that pooling doesn't actually improve performance anyway (in BYOND)
But I don't think anyone's actually tested it either way

Comment (N3X15) [reply code/game/objects/structures/grille.dm:42]: Pooling is suopposed to reduce memory usage, as well.

Comment (gbasood) [reply code/game/objects/structures/grille.dm:42]: Cluster keeps saying they tested it wrongly on tg when they removed it
Anyways, if you're gonna remove pooling, check that it doesn't hard del

Comment (Intigracy) [reply code/game/objects/structures/grille.dm:42]: >merely changing from getFromPool to new solved the issue

Obviously it's doing something differently

Comment (Exxion) [reply code/game/objects/structures/grille.dm:42]: Pooling causes several inscrutable issues

Comment (N3X15) [reply code/game/objects/structures/grille.dm:42]: Like?

Comment (Exxion) [reply code/game/objects/structures/grille.dm:42]: 1. This, somehow
2. Invisible grilles from the void

Alright, maybe not several, but at least two

**Comment by DamianX** [comment]
The conflicts are gone, by the way.

**Pull request merged**